.github/workflows/create-release-post.yml

```
@@ -0,0 +1,54 @@
+# Security Notes
+# Only selected Actions are allowed within this repository. Please refer to (https://github.com/nodejs/nodejs.org/settings/actions)
+# for the full list of available actions. If you want to add a new one, please reach out a maintainer with Admin permissions.
+# REVIEWERS, please always double-check security practices before merging a PR that contains Workflow changes!!
+# AUTHORS, please only use actions with explicit SHA references, and avoid using `@master` or `@main` references or `@version` tags.
+
+name: Create Release Blog Post
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: The version to generate a blog post for. If none provided, use the latest release.
+        type: string
+
+defaults:
+  run:
+    # This ensures that the working directory is the root of the repository
+    working-directory: ./
+
+permissions:
+  contents: write
+
+jobs:
+  create-post:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: nodejs/web-team/actions/setup-environment@2c2897a93eb99b4cdca270729100bc0887c758d9
+        with:
+          pnpm: true
+          use-version-file: true
+
+      - id: create-release-post
+        working-directory: apps/site
+        run: node --run scripts:release-post ${{ inputs.version }}
```

to protect against shell injection
```suggestion
        run: node --run scripts:release-post "$VERSION"
        env:
          VERSION: ${{ inputs.version }}
```

---

```suggestion
        run: echo "version=$(node --run scripts:release-post "$VERSION" --output-version)" >> "$GITHUB_OUTPUT"
        env:
          VERSION: ${{ inputs.version }}
```